csharp/private/actions/BUILD

```diff
@@ -0,0 +1,9 @@
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+
+package(default_visibility = ["//visibility:public"])
```

Comment:
I'm nervous about making our private stuff public (see the path)

Reply:
The usages of `bzl_library` are for aggregating rules files together, so that actions can be applied on them (like tests or documentation).  Setting the default visibility is a quick way of allowing this to work (without it we cannot find the '.bzl' files) for docs+tests. You can see a similar usage in [rules_python](https://github.com/bazelbuild/rules_python/blob/master/python/BUILD#L27)

I think we may be able to limit the visibility to `visibility = ["//docs:__pkg__"]` and use `exports_files` for exposing the rules files, but I'll need to delve into that a bit more first.

Reply:
After looking at how `bazel_tools` handles this, I think I may be able to use a combination of `filegroups` and restricted visibility to remove the need for the default visibility of public.

Reply:
This was successful at limiting visibility permissions, as well as cleaning up usages in all of the files.